src/lib/integrations/daoStack/arc/createDao.ts

```diff
@@ -74,19 +74,23 @@ export const createDao = async (
     cap
   )
 
-  const Avatar = await forgeOrg.call()
   let tx = await forgeOrg.send()
   console.log("Created new organization.")
   console.log(tx)
+  const Avatar = tx.events["NewOrg"].returnValues["_avatar"]
```

Comment:
This returns the same as `const Avatar = await forgeOrg.call()` does.

Reply:
I see that DAOstack has made the same change: https://github.com/daostack/migration/pull/116/files

Why is `const Avatar = await forgeOrg.call()` functioning for me??  😕 ❓

Reply:
I'm not sure...

Reply:
@sogasg I just tested this again in your branch and it's working... I'm not sure why though... it doesn't make sense to me that this code snippet works:
```
const Avatar = await forgeOrg.call()
let tx = await forgeOrg.send()
console.log(Avatar)
console.log(tx.events.NewOrg.returnValues._avatar)
```

What does `.call(...)` on a txMethod even mean? Why is it returning the correct result before the transaction has been sent? When doing `await forgeOrg.call()`, is it just running this on a single node "off chain" and returning back the result that should be?

Reply:
Regardless, not sure why this wasn't working on my machine last night, I must be loosing it... I'm okay with abandoning this PR if we can have some sort of answer around why .call works.